.clj-kondo/config/modules/config.edn

```diff
@@ -658,6 +659,7 @@
            metabase.parameters.field
            metabase.parameters.init
            metabase.parameters.params
+           metabase.parameters.schema
```

Comment:
Unfortunately `parameters` is too much of a mess to have everything go thru a `.core` namespace at this point in time. Not sure how we will square that with schemas in the future -- I'm thinking about making `.schema` the official fourth optional canonical API namespace. 

I was initially thinking about giving them a neutral keyword like `:parameters/` but it's actually important that namespace get loaded at compile time because `defendpoint` needs to resolve schemas when generating routes. So it has to be something that causes the namespace to get required.

Maybe `:metabase.parameters.core/<whatever>` would be better -- then everyone using them can require `metabase.parameters.core` -- I will revisit this in the future